core/lz4/src/ZipLZ4.cxx

```diff
@@ -0,0 +1,139 @@
+// Original Author: Brian Bockelman
+
+/*************************************************************************
+ * Copyright (C) 1995-2017, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include "ZipLZ4.h"
+#include "lz4.h"
+#include "lz4hc.h"
+#include <stdio.h>
+#include <cinttypes>
+#include <cstdint>
+#include <cstring>
+
+#include "RConfig.h"
+
+// Pulled from liblz4; upstream library explicitly exposes the symbol but the default build
+// excludes the header.
+typedef unsigned long long XXH64_hash_t;
```

Comment:
Not essential but I suppose the coding convention would say XXH64Hash_t or Xxh64Hash_t.